ui/src/test/org/jboss/windup/web/selenium/CreateProject.java

```diff
@@ -0,0 +1,1348 @@
+package org.jboss.windup.web.selenium;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.By.ByClassName;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.StaleElementReferenceException;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebDriverException;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.firefox.FirefoxDriver;
+import org.openqa.selenium.firefox.FirefoxOptions;
+//import org.openqa.selenium.remote.DesiredCapabilities;
+import org.openqa.selenium.htmlunit.HtmlUnitDriver;
+import org.openqa.selenium.support.ui.ExpectedCondition;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.openqa.selenium.firefox.GeckoDriverService;
+import org.jboss.windup.web.selenium.AnalyzeProject.Application;
+import org.openqa.selenium.Alert;
+import com.google.common.base.Preconditions;
+
+import java.awt.AWTException;
+import java.awt.Robot;
+import java.awt.event.KeyEvent;
+import java.io.File;
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.util.*;
+import java.util.logging.Level;
+
+/**
+ * this code is intended for a RHAMT web application that does not have any
+ * pre-made projects
+ * 
+ * @author elise
+ *
+ */
+public class CreateProject {
+
+	private WebDriver driver;
+
+	private WebElement projButton;
+	private WebElement nameInput;
+	private WebElement descInput;
+	private WebElement cancel;
+	private WebElement next;
+	private WebElement chooseFiles;
+	private WebElement fileUpload;
+	private WebElement checkbox;
+
+	public CreateProject() {
+		// Create a new instance of the Firefox driver
+		// Notice that the remainder of the code relies on the interface,
+		// not the implementation.
+		System.setProperty("webdriver.gecko.driver", "/usr/lib/node_modules/geckodriver/bin/geckodriver");
+		System.setProperty("webdriver.chrome.driver","/usr/bin/chromedriver");
+
+		ChromeOptions options = new ChromeOptions();
+		options.setBinary("/usr/bin/chromium-browser"); // Location where Chrome is installed
```

Comment:
This is not going to work with Chrome ;)
Better comment this line (and all the other occurrences) and put ["standard" Chrome installation](https://github.com/SeleniumHQ/selenium/wiki/ChromeDriver#requirements) as pre-requisite